### PR TITLE
Support skip auth option in get-file

### PR DESCRIPTION
**Pull request by raaaaaaaaaan**

### CODE REVIEW

File: src/http/get-file.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ import compressFile from './compress-file';
 
 export default opts => {
   const { storage, fileKey, req, res, contentType, auth } = opts;
-
   auth(err => {
-    const acl = err ? 'public' : 'private'; // if auth fails, pass as public request
+    const acl = err && !auth.skip ? 'public' : 'private'; // if auth fails, pass as public request
```
Comment (asilvas) [inline src/http/get-file.js:7]: `err || auth.skip === true`

File: src/http/get-auth.js [hunk 2]
```diff
@@ -33,5 +32,6 @@ function getAuthHandler(req, fileKey, storageId, authId, authConfig, driver) {
     driver(req, storageId, fileKey, authConfig.options || {}, cb);
   };
   handler.id = authId;
+  handler.skip = authConfig.options.skipAuth;
```
Comment (asilvas) [inline src/http/get-auth.js:35]: `handler.skip = authConfig.options.skipAuth === true`

File: src/http/get-file.js (hunk 1 above)
Comment (raaaaaaaaaan) [reply src/http/get-file.js:7]: If `auth.skip === true`, we wanna ignore `auth err` and create knox client to get a photo

Comment (asilvas) [reply src/http/get-file.js]: Which my code will do.

**Comment by raaaaaaaaaan** [comment]
According to discussion with Aaron, our photos (<1 millions files) don't need to use blobby for now

**Pull request closed without merging**